superset/databases/commands/create.py

```diff
@@ -33,6 +33,7 @@
 from superset.databases.commands.test_connection import TestConnectionDatabaseCommand
 from superset.databases.dao import DatabaseDAO
 from superset.extensions import db, event_logger, security_manager
+from superset.models.core import ConfigurationMethod
```

Comment:
is this used anywhere here?

Reply:
No it used to be, and I forgot to take it out.